gravitee-apim-console-webui/src/services-ngx/api-v2.service.ts

```diff
@@ -128,7 +128,7 @@ export class ApiV2Service {
 
   getLastApiFetch(apiId: string): Observable<Api> {
     if (this.lastApiFetch$.value === null) {
-      return this.get(apiId);
+      this.get(apiId).subscribe();
```

Comment:
I would like to discuss this fix before merging please

Reply:
@Okhelifi as discussed with you, I've pushed with the suggestion of @loriepisicchio